project1/css/index.css

```diff
@@ -62,3 +62,103 @@ h1, h2, h3, h4, h5 {
     letter-spacing: 1px;
     margin-bottom: 10px;
 }
+a{
+	color:gray;
+	text-decoration:none;
+}
+ul{
```

Comment:
`ul {

}`

Let's try to keep the spacing around the target element and curly brace consistent throughout the code. The way you want to write code is completely fine, and style suggestions are just that. But when writing code in your chosen style make sure to keep it consistent for nice clean code! :)